progenitor-impl/tests/output/buildomat-builder-tagged.out

```diff
@@ -2355,6 +2367,51 @@ pub mod builder {
         }
     }
 
+    ///Builder for [`Client::whoami_put_name`]
+    ///
+    ///[`Client::whoami_put_name`]: super::Client::whoami_put_name
+    #[derive(Debug)]
+    pub struct WhoamiPutName<'a> {
+        client: &'a super::Client,
+        body: Result<reqwest::Body, String>,
+    }
+
+    impl<'a> WhoamiPutName<'a> {
+        pub fn new(client: &'a super::Client) -> Self {
+            Self {
+                client,
+                body: Err("body was not initialized".to_string()),
+            }
+        }
+
+        pub fn body(mut self, value: String) -> Self {
```

Comment:
In other situations where there's a string parameter we have code like this:
```rust
        pub fn task<V>(mut self, value: V) -> Self
        where
            V: std::convert::TryInto<String>,
        {
```

Should we do something similar here rather than just an explicit `String`?

Reply:
Yeah, that sounds reasonable! I've changed the builder implementation now to look like this.